docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,162 @@
+= Documentation for core maintainers
+
+Jenkins core is a central component of the project which serves millions of users, and ot is critical to maintain the it in a good shape.
+We want to ensure quality of the integrated changes and continuity of the project,
+and hence the Jenkins core pull request review and merge process is more sophisticated than for the majority of plugins.
+
+== Scope of the document
+
+This document applies to the following components:
+
+* Jenkins core
+* Jenkins modules
+* Libraries included into the Jenkins core
+* Core components like Winstone, Extras Executable WAR, etc.
+
+Jenkins Remoting library is not in the scope for this document,
+because it is handled as a sub-project with independent maintainer team.
+Remoting updates in the core are subject to the process though.
+
+== Team
+
+=== Communication channels
+
+* link:https://groups.google.com/d/forum/jenkinsci-dev[Jenkins Developer Mailing List]
+* Chat: N/A, to be created
+* Office hours: N/A, to be created
```

Comment:
No need to mention this or chat.

Reply:
I would prefer to do it so that we add contacts in the nearest future

Reply:
It can be added when they are created?

Reply:
```suggestion
* Office hours: N/A, to be created
```

Reply:
```suggestion
```